docs/user/install-gke.md

```diff
@@ -4,88 +4,31 @@
-```
-gcloud container clusters create CLUSTER_NAME \
-    --release-channel None \
-    --zone COMPUTE_ZONE \ # Should match GCS/GCP Zones.
-    --node-locations COMPUTE_ZONE # Should match GCS/GCP Zones.
-    --num_nodes 3 \ # Change based on processing requirements.
-    --machine-type e2-medium # Change based on processing requirements.
-    --scopes "https://www.googleapis.com/auth/cloud-platform" 
-```
-### **Turbinia GKE Deployment**
-* Connect to cluster through `gcloud container clusters get-credentials <CLUSTER_NAME> --zone <ZONE> --project <PROJECT_NAME>`.
-* Clone the latest Turbinia branch and `cd <git clone path>/k8s/gcp-pubsub`.
-* Ensure that the zone and region in the Turbinia config file are equal to the zone and region you created your k8s cluster in.
-* The `image` variable can be optionally changed in the `turbinia-worker.yaml` and `turbinia-server.yaml` files to chose the docker images used during deployment.
-* In the `turbinia-worker.yaml` file, ensure that the path in the volume labeled `lockfolder` matches the Turbinia config variable `TMP_RESOURCE_DIR`.
-* If the Filestore `Instance ID` and `File share name` have a different name than the default name `output`, update `turbinia-worker.yaml`, `turbinia-server.yaml`, `turbinia-output-claim-filestore.yaml`, and `turbinia-output-filestore.yaml` by searching for the string `output` and replacing it with the custom name. Skip this step if the default name was used.
-* To have all logs go to the central location, update the `LOG_DIR` variable in the `.turbiniarc` config file to the default Filestore path `/mnt/output` or if configured differently, to the custom path.
-* In `turbinia-output-filestore.yaml`, update `<IP_ADDRESS>` to the Filestore IP address.
-* If the Filestore instance size is greater than 1 TB, update the `storage` sections in `turbinia-output-claim-filestore.yaml` and `turbinia-output-filestore.yaml` with the appropriate size.
-* Ensure that the `.turbiniarc` config file has been properly configured with required GCP variables.
-* Deploy the Turbinia infrastructure by executing `./setup-pubsub.sh <PATH TO CONFIG>`. 
-* The Turbinia infrastructure can be destroyed by executing `./destroy-pubsub.sh`.
+* Review the `.clusterconfig` config file and please update any of the default values if necessary based on requirements.
+* Deploy Turbinia through the following command
+    * `./k8s/tools/deploy-pubsub-gke.sh`
+* The deployment script will automatically enable GCP APIs, create the cluster and GCP resources then deploy Turbinia to the cluster. At the end of the run, you should have a fully functioning Turbinia environment within GKE to use. 
+* Run the following command if you'd like to cleanup the newly created Turbinia environment
+    * `./k8s/tools/cleanup-pubsub-gke.sh`
+    * **Note this will delete the Turbinia cluster and associated GCP resources** 
```

Comment:
Maybe explicitly add "including all processed output and log files"

Reply:
Done